File: MODUL-1/C/PRAK104-2310817220002-RaudatulSholehah.c
#include <stdio.h>
int main()
{
    int harga_sepatu_a;
    printf("Harga sepatu A adalah: ");
    scanf("%d", &harga_sepatu_a);
    int harga_sepatu_b;
    printf("Harga sepatu B adalah: ");
    scanf("%d", &harga_sepatu_b);

    float harga_sepatu_a_diskon = 0.13;
    float harga_sepatu_b_diskon = 0.21;

    printf("Sepatu A mendapat diskon 13%% sehingga harganya menjadi: %.0f\n", harga_sepatu_a - (harga_sepatu_a * harga_sepatu_a_diskon));
    printf("Sepatu B mendapat diskon 21%% sehingga harganya menjadi: %.0f\n", harga_sepatu_b - (harga_sepatu_b * harga_sepatu_b_diskon));

    return 0;
}

File: MODUL-1/Python/PRAK104-2310817220002-RaudatulSholehah.py
harga_sepatu_a = int(input("Harga sepatu A adalah: "))
harga_sepatu_b = int(input("Harga sepatu B adalah: "))
harga_sepatu_a_diskon = 0.13
harga_sepatu_b_diskon = 0.21

print(f"Sepatu A mendapat diskon 13%% sehingga harganya menjadi: {harga_sepatu_a - (harga_sepatu_a * harga_sepatu_a_diskon):.0f}")
print(f"Sepatu B mendapat diskon 21%% sehingga harganya menjadi: {harga_sepatu_b - (harga_sepatu_b * harga_sepatu_b_diskon):.0f}")